deployment/chainloop/templates/cas/deployment.yaml

```diff
@@ -71,7 +71,7 @@ spec:
           {{- include "common.tplvalues.render" (dict "value" .Values.cas.initContainers "context" $) | nindent 8 }}
         {{- end }}
       containers:
-        - name: {{ .Chart.Name }}
+        - name: cas
```

Comment:
what about cas-proxy? Could it be changed without breaking the tests? How hard would it be to change those?

Reply:
We would have to update the tests since the name of the container is hardcoded:

```golang
By("checking main container image is running")
_, err = utils.DplGetContainerImage(dpl, "cas")
Expect(err).NotTo(HaveOccurred())
```

Reply:
is it hard to change?

Reply:
No, it should be only changing that line to `cas-proxy` and open another PR for the Bitnami team

Reply:
Change in Bitnami: https://github.com/bitnami/charts/pull/28830